tests/DI/DoctrineBehaviorsExtensionTest.php

```diff
@@ -23,7 +23,7 @@ class DoctrineBehaviorsExtensionTest extends PHPUnit_Framework_TestCase
 	/**
 	 * @var int
 	 */
-	const LISTENER_COUNT = 18;
+	const LISTENER_COUNT = 19;
```

Comment:
Actually this was changed in the master branch already, see https://github.com/Zenify/DoctrineBehaviors/commit/ead0c88ea20a2dc9d1d3153207873c1a2a036fc7.

Please rebase this PR to the current master.